genet/use/cli.py

```diff
@@ -0,0 +1,951 @@
+import json
```

Comment:
A question about naming...

It doesn't seem like the code in this module is tied to the CLI. If I were using GeNet from a notebook, an app with a web front end, or a Python script, (i.e. *not* the CLI) I would be using many of these functions to do things like read or write a network, generate Google Directions API requests, etc.

In [MVC](https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller) terms, this module is arguably a mixture of **controller** and **model**, but it isn't the **view**/bit used directly by the command line user.

This refactoring is about the separation of concerns, which I like. We've made the CLI code a thin layer around the "business logic" of the library, responsible only for figuring out what the command line user wants to do and translating that into operations on the core library. Good stuff.

But the code in this module has now (rightly) been relieved of all CLI responsibility and represents "business logic", so the name `cli` doesn't seem right to me. There isn't actually any CLI code in here, as far as I can see, and there's no reason this module should be/considered to be for the exclusive use of the CLI.

Can we come up with a better, more descriptive name?

Reply:
I'm leaning towards `recipes` - does that sound good?